raiden/api/python.py

```diff
@@ -461,25 +481,25 @@ def set_total_channel_deposit(
             channel_id=channel_state.identifier,
         )
 
-        balance = token.balance_of(self.raiden.address)
+        total_network_balance = token.balance_of(registry_address)
 
-        if self.raiden.config['environment_type'] == Environment.PRODUCTION:
-            deposit_limit = (
-                token_network_proxy.proxy.contract.functions.
-                channel_participant_deposit_limit().call()
-            )
-        elif self.raiden.config['environment_type'] == Environment.DEVELOPMENT:
-            deposit_limit = (
-                token_network_proxy.proxy.contract.functions.
-                deposit_limit().call()
+        if total_network_balance + total_deposit > per_token_network_deposit_limit:
+            raise DepositOverLimit(
+                f'The deposit of {total_deposit} exceeds '
+                f'token network limit of {per_token_network_deposit_limit}',
             )
 
+        balance = token.balance_of(self.raiden.address)
+
+        deposit_limit = (
+            token_network_proxy.proxy.contract.functions.
+            channel_participant_deposit_limit().call()
+        )
+
         if total_deposit > deposit_limit:
             raise DepositOverLimit(
-                'The deposit of {} is bigger than the current limit of {}'.format(
-                    total_deposit,
-                    deposit_limit,
-                ),
+                f'The deposit of {total_deposit} is bigger than the current '
```

Comment:
Use addendum and also reword the error:  The addition of addendum to the channel by the participant will exceed the participant limit.